semantic/mesh/roof/mesh-repair.cpp

```diff
@@ -25,12 +25,17 @@
  */
 
 #include <algorithm>
+#include <queue>
 
 #include "dbglog/dbglog.hpp"
 #include "geometry/mesh.hpp"
 #include "geometry/meshop.hpp"
 
+#include "../../mesh.hpp"
+#include "../detail.hpp"
+
 #include "math/geometry.hpp"
+#include "vadstena-libs/meshio.hpp"
```

Comment:
I guess this obsolete include of an internal library slipped through. I hope it's not used anywhere - please check and fix @katarinagresova